Tools/GraphGeneration/examples/config_files/config.xml

```diff
@@ -36,28 +36,28 @@
       <AvgDrivingSpeed name="AverageDrivingSpeed">30.0</AvgDrivingSpeed>
     </VerticesParams>
 
-    <EdgesParams class="AllGraphGenerationEdges" name="EdgesParams">
+    <EdgesParams class="All911Edges" name="EdgesParams">
     </EdgesParams>
 
-    <ConnectionsParams class="ConnectionsGraphGeneration" name="ConnectionsParams">
+    <ConnectionsParams class="Connections911" name="ConnectionsParams">
     <!-- Graphml file for graph modeling details -->
-      <graphmlFile name="graphmlFile">../GraphGeneration/synth_input.graphml</graphmlFile>
+      <graphmlFile name="graphmlFile">/home/NETID/jsaini/Graphitti/Tools/GraphGeneration/examples/graphml_generation/synth_input.graphml.xml</graphmlFile>
```

Comment:
Make this a relative path, rather than an absolute path. It should work for anyone.